src/Integration.Vsix/ErrorList/SonarErrorListDataSource.cs

```diff
@@ -222,6 +237,11 @@ public void Refresh(IEnumerable<string> affectedFilePaths)
                     {
                         factory.UpdateSnapshot(factory.CurrentSnapshot.GetUpdatedSnapshot());
                         InternalRefreshErrorList(factory);
+
+                        if (notifyListeners)
+                        {
+                            NotifyIssuesChanged(factory);
+                        }
```

Comment:
This is a simple solution that might result in duplicate notifications in rare cases involving secondary locations
e.g.
 * Factory1 has Issue1 that refers to FileA and FileB
 * Factory2 has Issue2 that refers to FileC and FileB

In that case the tagger for FileB will be notified twice.
I'll add a TODO and a separate ticket.